src/network/task.rs

```diff
@@ -380,26 +412,31 @@ impl NetworkTask {
             #[cfg(debug_assertions)]
             log::debug!("Waiting for events ...");
 
+            if send_py_permit.is_none() {
+                send_py_permit = self.py_tx.try_reserve().ok();
+            }
+            let can_send_net = self.net_tx.capacity() > 0;
```

Comment:
Same question here: Weird name for this variable. For example, using `net_tx_full` and inverting the logic would be more clear.

Reply:
Happy with either! :)